### PR TITLE
highlights for week ending August 19, 2022

### PR DESCRIPTION
**Pull request by gianfranco-l**
### Please remember to

- [ ] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/changes/highlights/2022-08-19.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+## General
+
+
+## User interfaces
+
+
+## Apps
+
```
Comment (stone-z) [inline src/content/changes/highlights/2022-08-19.md:17]:
```suggestion

- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.11.0](https://github.com/giantswarm/kyverno-app/blob/master/CHANGELOG.md#0110---2022-08-17) updates to upstream version 1.7.2, containing several resilience and performance improvements.
- [security-pack](https://github.com/giantswarm/security-pack) version [v0.6.0](https://github.com/giantswarm/security-pack/blob/main/CHANGELOG.md#060---2022-08-17) includes the latest Kyverno app release (v0.11.0) as well as minor updates to Starboard exporter (v0.5.1) and Trivy (v0.28.1).
```

**Review by stone-z — APPROVED** [review]

**Comment by gianfranco-l** [comment]
as lucas is AFK, pinging @ubergesundheit @ced0ps  for cabbage and @QuentinBisson @TheoBrigitte for atlas : anything to announce to customers this week from your team? Contributions accepted until Friday 12pm CET :)

**Comment by alex-dabija** [comment]
No updates from Hydra and Phoenix.

**Comment by Gacko** [comment]
From the top off my head I only remember a `nginx-ingress-controller-app` patch release. No changes in `cert-manager-app` or `external-dns-app`. And I'm probably even worse at remembering all the others...

File: src/content/changes/highlights/2022-08-19.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+
+## User interfaces
+
+
```
Comment (marians) [inline src/content/changes/highlights/2022-08-19.md:15]:
```suggestion
## User interfaces

We fixed a problem in our web UI that has prevented some non-admin users from logging in. In addition, in the list of apps installed in a cluster, it is now easier to spot the ones that are not in the expected "Deployed" state.

```

Comment (marians) [reply src/content/changes/highlights/2022-08-19.md:15]: @gianfranco-l This is team Rainbow's only contribution this week.

**Pull request merged**